### PR TITLE
Feature/move disconnect button to page header

### PR DESCRIPTION
**Pull request by gabrielHosino**
#### What is the purpose of this pull request?

This PR makes an UI change, we moved the disconnect button to the page header. Besides we took this oportunity to refactor the header to use the admin-ui components

#### How to test it?

You can see the changes in this [workspace](https://tiktok--gabrielhosino.myvtex.com/admin/tiktok)

#### Screenshots or example usage
image

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Move disconnect button to page header
+- Refactor header to use admin-ui components
```
Comment (felipefadul) [inline CHANGELOG.md:13]:
```suggestion
- Move disconnect button to page header.
- Refactor header to use admin-ui components.
```

**Review by felipefadul — APPROVED** [review]
LGTM!

File: react/components/TikTokHeader.tsx [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
-    <div className={tikTok.headerBar}>
-      <div>
+    <PageHeader>
+      <PageTitle>
         <img alt="logo" className={tikTok.logo} src={TBP_LOGO} />
-        <span>For Business</span>
-      </div>
-    </div>
+        <Text csx={{ marginLeft: 0 }} variant="title2">
+          For Business
```
Comment (anamaiabap) [inline react/components/TikTokHeader.tsx:19]: Não sei se já está usando a internacionalização, mas se está, faltou aqui 🙈

Comment (gabrielHosino) [reply react/components/TikTokHeader.tsx:19]: Oi Ana, nesse caso não é necessário a internacionalização. porque TikTok For Business é o nome do produto.

Comment (anamaiabap) [reply react/components/TikTokHeader.tsx:19]: Aaaa
Perfeito então!!
Vou aprovar aqui :)

**Review by anamaiabap — APPROVED** [review]

**Pull request merged**